rosdep/base.yaml

```diff
@@ -6842,6 +6842,16 @@ muparser:
   nixos: [muparser]
   opensuse: [muparser-devel]
   ubuntu: [libmuparser-dev]
+nanoflann:
```

Comment:
I just noticed that for these kinds of keys, we usually follow the Ubuntu naming policy for it.  So I think this should be `libnanoflann-dev`.

Reply:
Yep, this key should be renamed to match the Debian/Ubuntu package name.
That will also require it to change locations in the file which is lexicographically sorted (otherwise I would add it as a suggestion to this comment).